docs/actions/comment.rst

```diff
@@ -7,6 +7,7 @@ Comment
       payload:
         body: >
           Your very long comment can go here.
+      old_comment: 'delete' # delete by default, if you wish to leave the old comments use 'leave'
```

Comment:
This looks more like a boolean. do you envision there be more options here? And if so would it make more sense for those to be options?

Reply:
I imagine, there could be either `edit` or `overwrite` option (not decided on the naming yet), similar to how code-cov does it, it maybe helpful to enforce rules on issues without too much noise.

Reply:
I see your point, i'll change it into a boolean